### PR TITLE
Revision of snapshotTime handling in HLT and new Run2 MC GT with ES fixes

### PR DESCRIPTION
**Pull request by diguida**
# Summary of Changes for HLT Testing

Revision of snapshotTime handling in HLT: no longer use/set a global `snapshotTime` but move to a per-payload `snapshotTime`, set to the time of PR #12095 
# Summary of Changes in the GT

Originally in #12114 commit https://github.com/mmusich/cmssw/commit/bee66c2e63f42f434e9cd95740d37c49a7e6ffef
## RunII Simulation
- RunII Ideal scenario : 76X_mcRun2_design_v11 as 76X_mcRun2_design_v10 with the following changes:
  - updated Preshower pedestals as per pedestal run in low gain taken in Run2015D
  - updated Preshower Zero suppression thresholds with 6 ADC
- RunII Asymptotic scenario : 76X_mcRun2_asymptotic_v11 as 76X_mcRun2_asymptotic_v10 with the following changes:
  - updated Preshower bad channels as per Run2015D
  - updated Preshower pedestals as per pedestal run in low gain taken in Run2015D
  - updated Preshower Zero suppression thresholds with 6 ADC
- RunII Startup scenario : 76X_mcRun2_startup_v11 as 76X_mcRun2_startup_v10 with the following changes:
  - updated Preshower bad channels as per Run2015D
  - updated Preshower pedestals as per pedestal run in low gain taken in Run2015D
  - updated Preshower Zero suppression thresholds with 6 ADC
- RunII Heavy Ions scenario : 76X_mcRun2_HeavyIon_v11 as 76X_mcRun2_HeavyIon_v10 with the following changes:
  - updated Preshower bad channels as per Run2015D
  - updated Preshower pedestals as per pedestal run in low gain taken in Run2015D
  - updated Preshower Zero suppression thresholds with 6 ADC

### CODE REVIEW

**Comment by diguida** [comment]
please test

**Comment by diguida** [comment]
@cmsbuild please acknowledge changes

**Comment by diguida** [comment]
please test

**Comment by diguida** [comment]
@cmsbuild please acknowledge changes

**Comment by diguida** [comment]
please test

File: Configuration/HLT/python/autoCondHLT.py [hunk 1]
```diff
@@ -1,24 +1,30 @@
 # https://cms-conddb.cern.ch/browser/#search
 
+#default value for all L1T menus
+connectionString = "frontier://FrontierProd/CMS_CONDITIONS"
+l1MenuRecord = "L1GtTriggerMenuRcd"
+l1MenuLabel = ""
+
+#The snapshot time has been set as starting point as the one of PR 12096.
```
Comment (Martin-Grunewald) [inline Configuration/HLT/python/autoCondHLT.py]: This should be 12095 (not 12096)...

**Comment by diguida** [comment]
@cmsbuild please acknowledge changes
Amended commits for fixing the typo noted by @Martin-Grunewald and for making the bot happy.

**Comment by diguida** [comment]
please test

**Comment by diguida** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1

**Pull request merged**